components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/scope/RoleBasedScopeIssuer.java

```diff
@@ -102,11 +102,15 @@ public class RoleBasedScopeIssuer extends AbstractRoleBasedScopeIssuer implement
     private static final String ISSUER_PREFIX = "default";
     OAuthServerConfiguration oAuthServerConfiguration = OAuthServerConfiguration.getInstance();
     private static final String REFRESH_TOKEN_GRANT_TYPE = "refresh_token";
+    private static final String ROLE_BASE_SCOPE_VALIDATION_ENABLED = "OAuth.RoleBasedScopeValidation.Enable";
```

Comment:
```suggestion
    private static final String ROLE_BASED_SCOPE_ISSUER_ENABLED = "OAuth.RoleBasedScopeValidation.Enable";
```

Reply:
Let's stick into the same name to avoid confusions with default validators

Reply:
Resolved with 3bd459952a79f935d5fac691386ba3f3489200c9